docs/architecture/adr-112-proposer-based-timestamps.md

```diff
@@ -292,17 +305,31 @@ The only change we will make to the prevote step is to what a validator consider
 The precommit step will not require much modification.
 Its proposal validation rules will change in the same ways that validation will change in the prevote step with the exception of the `timely` check: precommit validation will never check that the timestamp is `timely`.
 
+<!---
 ### Remove voteTime Completely
 
-[voteTime](https://github.com/cometbft/cometbft/blob/822893615564cb20b002dd5cf3b42b8d364cb7d9/internal/consensus/state.go#L2229) is a mechanism for calculating the next `BFTTime` given both the validator's current known Unix time and the previous block timestamp.
+[voteTime](https://github.com/cometbft/cometbft/blob/822893615564cb20b002dd5cf3b42b8d364cb7d9/internal/consensus/state.go#L2229) is a mechanism for calculating the next `BFT Time` given both the validator's current known Unix time and the previous block timestamp.
 If the previous block timestamp is greater than the validator's current known Unix time, then voteTime returns a value one millisecond greater than the previous block timestamp.
-This logic is used in multiple places and is no longer needed for proposer-based timestamps.
+This logic is used in multiple places and is no longer needed for PBTS.
 It should therefore be removed completely.
+--->
+
+### Backwards compatibility
+
+In order to ensure backwards compatibility, PBTS should be enabled using a consensus parameter.
+The proposed approach is similar to the one adopted to enable vote extensions:
+[`ABCIParams.VoteExtensionsEnableHeight`](https://github.com/cometbft/cometbft/blob/main/spec/abci/abci++_app_requirements.md#abciparamsvoteextensionsenableheight).
+
+In summary, the network will migrate from the `BFT Time` method for assigning
+and validating timestamps to the new method for assigning and validating
+timestamps adopted by `PBTS` from a given, configurable height.
+
+For more discussion of this, see [issue 2063][issue2063].
 
 ## Future Improvements
 
 * Implement BLS signature aggregation.
-By removing fields from the `Precommit` messages, we are able to aggregate signatures.
+If we remove the `Timestamp` field from the `Precommit` messages, we are able to aggregate signatures.
```

Comment:
```suggestion
If we remove the `Timestamp` field from the `Precommit` messages, we are able to aggregate signatures.
We have however left this improvement out for the time being, as it would break the block format and validation
rules (signature verification) and thus may force a hard-fork on chains upgrading to the latest version of CometBFT.
We will remove the timestamps in votes when changing the block format is supported in CometBFT without
requiring a hard-fork (this feature is called Soft Upgrades).
```

Reply:
I hate merging stuff which prevents accepting suggestions. Added this change in a new commit.